group_vars/all/main.yml

```diff
@@ -82,11 +82,12 @@ richie_database:
 
 # Openshift template files
 openshift_deployments:
+  - openshift/common/dc/elasticsearch.yml
```

Comment:
In fact it's not common because richie and edxapp require different versions of ES. So this one is dedicated to Richie.

Reply:
True. For now, I will make this dc/svc richie-specific. We will add ES for edxapp later.

Reply:
I think we have to abandon the `common` directory.
I think we should consider the `templates/openshift` directory as a collection of usable micro-services for an application.
I propose that we adopt the following naming convention:
`templates/openshift/<micro-service-name>/{svc,dc,pcv,...}`

If we do like this in the future we can find all templates use by a microservice and not hardcode the path in an ansible vars.

Reply:
Agree! I'll switch to this pattern next week.